package.json

```diff
@@ -308,6 +308,11 @@
                                     "normal"
                                 ]
                             },
+                            "injectTrackerTask": {
```

Comment:
I think we should include RALE somewhere in this name. something like `injectRaleTrackerTask`? Thoughts?